### DISCUSSION TITLE
Add a framework to the tests

### DISCUSSION THREAD

**Discussion by amirfaramarzi** [TechEmpower/FrameworkBenchmarks#7904] [General]
Hi everyone

I am an Iranian, as you probably know, the government has filtered the country's Internet very widely, and I have been trying to add my own framework for a few days, but I am having problems due to the very severe filtering (Docker does not work, packages are not downloaded. And.... ,which has messed up my nerves)

My question is, can anyone add my framework to techempower tests???

If you can help me, please send your telegram id

**Comment by joanhey** [comment]
Please write the language and framework name.
So will be easier to find anybody to help you.

**Comment by amirfaramarzi** [comment]
hi @joanhey 

The framework is written in PHP and is called fomo

GitHub framework link: https://github.com/fomo-framework/fomo

**Comment by joanhey** [comment]
I will help you.
Git me some days.

**Reply by amirfaramarzi** [reply]
thank you very much

**Comment by joanhey** [comment]
Fomo added
https://github.com/TechEmpower/FrameworkBenchmarks/pull/9584

**Comment by joanhey** [comment]
Please fix the deprecated errors with PHP8.4

https://github.com/joanhey/FrameworkBenchmarks/actions/runs/13224796813/job/36914299789#step:9:139
